CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 [//]: # (current developments)
 
+## 24.1.0 (2024-01-24)
+
+### Enhancements
+
+* Verify signatures on to-be-installed packages instead of on all packages. (#11545, #13053)
+* Add a new `pre-solves` plugin hook. (#13053)
+* Add a new `post-solves` plugin hook. (#13053)
+* Add support for Python 3.12. (#13072)
+* Check `repodata.json.zst` for faster repodata downloads. (#13256)
+* Add `--skip-flexible-search` option to skip flexible search. (#13315)
+* Add type hinting for `VersionOrder` class. (#13380)
+* Provide a more useful warning when attempting to rename a non-existent prefix. (#13387)
+* Add a new flag `--keep-env` to be used with `conda remove --all`. It allows users to delete all packages in the environment while retaining the environment itself. (#13419)
+* Add a Y/N prompt warning users that `conda env remove` and `conda remove --all` deletes not only the conda packages but the entirety of the specified environment. (#12222 via #13440)
+* Add `--no-repodata-zst` flag to skip `repodata.json.zst` check. Useful if the repo returns unparseable `repodata.json.zst` instead of correct data or 404. (#13504)
```

Comment:
What's the equivalent option in `.condarc`? Out of curiosity.

Reply:
@dholth Do you know?

Reply:
It looks like we need to add the condarc option which would have to be `no_repodata_zst: true`

I also noticed a `repodata_use_zst: true` in my condarc which is a mamba option.

Reply:
Sounds like something to fix in 24.1.1? Or blocking the release?

Reply:
https://github.com/conda/conda/issues/13528

Reply:
Thank you @dholth

Reply:
Oh, the environment variable works in the test suite https://github.com/conda/conda/blob/main/tests/gateways/test_jlap.py#L371-L378 but for some reason it isn't working from the command line.

Reply:
Yes, that was the issue. See https://github.com/conda/conda/pull/13530